---
 CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 44d1162c..35a991fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,7 +26,6 @@ endif()
 
 include(ExternalProject)
 include(FeatureSummary)
-include(FetchContent)
 
 set(ECM_MODULE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/cmake/ECM/modules/")
 list(APPEND CMAKE_MODULE_PATH ${ECM_MODULE_DIR} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/ECM/kde-modules" )
@@ -37,13 +36,6 @@ include(KDEFrameworkCompilerSettings NO_POLICY_SCOPE)
 include(ECMQtDeclareLoggingCategory)
 include(ECMInstallIcons)
 
-FetchContent_Declare(
-    KDExtraCMakeModules
-    GIT_REPOSITORY ssh://codereview.kdab.com:29418/kdab/extra-cmake-modules
-    GIT_TAG master
-    SOURCE_DIR ${CMAKE_SOURCE_DIR}/cmake/KDExtraCMakeModules
-)
-FetchContent_MakeAvailable(KDExtraCMakeModules)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/KDExtraCMakeModules/modules")
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
-- 
2.40.1

