---
 makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/makefile b/makefile
index f8cc6ed..1c20a43 100644
--- a/makefile
+++ b/makefile
@@ -1,9 +1,6 @@
 NAME = ly
 CC = gcc
-FLAGS = -std=c99 -pedantic -g
-FLAGS+= -Wall -Wextra -Werror=vla -Wno-unused-parameter
-#FLAGS+= -DDEBUG
-FLAGS+= -DGIT_VERSION_STRING=\"$(shell git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g')\"
+FLAGS= -DGIT_VERSION_STRING=\"$(shell git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g')\"
 LINK = -lpam -lxcb
 VALGRIND = --show-leak-kinds=all --track-origins=yes --leak-check=full --suppressions=../res/valgrind.supp
 CMD = ./$(NAME)
-- 
2.38.3

